File: algorithms_organized_in_pictures/chap04/s8_Queue/Queue.py
# 파이썬으로 구현한 큐 자료구조
def Queue():
    queue = []
    queue.append(1)
    queue.append(2)
    queue.append(3)
    queue.append(4)
    queue.append(5)
    print(queue)

    while queue:
        print("Get Value:", queue.pop(0))

Queue()

File: algorithms_organized_in_pictures/chap04/s8_Queue/Queue.java
// 자바로 구현한 큐 자료구조
import java.util.*;

public class Queue {
    public static void main(String[] args) {

        LinkedList<String> ls = new LinkedList<String>();

        ls.offer("홍길동");    // 큐에 저장
        ls.offer("김동자");
        ls.offer("유명한");
        ls.offer("천세원");

        System.out.println("Size = " + ls.size());
        while (ls.peek() != null)   // 읽을 값이 있는가?
        System.out.println(ls.poll());  // 값을 읽어서 출력

        System.out.println("Size = " + ls.size());
    }
}
